src/Induction/WellFounded.agda

```diff
@@ -116,6 +119,21 @@ module FixPoint
   unfold-wfRec : ∀ {x} → wfRec P f x ≡ f x λ _ → wfRec P f _
   unfold-wfRec {x} = f-ext x wfRecBuilder-wfRec
 
+------------------------------------------------------------------------
+-- Well-founded relations are asymmetric and irreflexive.
+
+module _ {_<_ : Rel A r} where
+  acc-asym : ∀ {x y} → Acc _<_ x → x < y → ¬ (y < x)
+  acc-asym {x} {y} hx = 
+    (Some.wfRec _ λ x hx y x<y y<x → hx y y<x x y<x x<y) x hx y
```

Comment:
I calculate the following to repair things after the merge conflict:
```agda
acc-asym {x} hx = Some.wfRec (λ x → ∀ {y} → x < y → ¬ (y < x)) (λ x hx x<y y<x → hx y<x y<x x<y) x hx
```
where now the explicit predicate argument needs to be supplied to `Some.wfRec`, and the implicit arguments `y` are suppressed.

HTH.

Reply:
Thank you for solving this problem, and sorry for the late response. I wasn't sure what's the correct way to get the new commits into my branch - I'm still a bit new to git.